rpcd/playbooks/roles/rpc_setup_infrastructure/tasks/sshd.yml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
Possibly, if we were fine with running the support role on it's own, and before anything else is laid down as well.